daliuge-engine/test/manager/test_daemon.py

```diff
@@ -143,7 +143,7 @@ def test_zeroconf_discovery(self):
         # if we query the MM it should know about its nodes, which should have
         # one element
         mc = MasterManagerClient()
-        nodes = _get_nodes_from_client(mc)
+        nodes = _get_nodes_from_client(mc)["nodes"]
```

Comment:
I stumbled upon this unittest debugging something else and it turns out to have not been testing the correct thing. 

This addition is in-line with other tests in the file.